specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -3589,6 +3589,60 @@
         },
         "x-ms-long-running-operation": true
       },
+      "patch": {
+        "tags": [
+          "VirtualMachineScaleSetExtensions"
+        ],
+        "operationId": "VirtualMachineScaleSetExtensions_Update",
+        "description": "The operation to update an extension.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmScaleSetName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VM scale set where the extension should be updated."
+          },
+          {
+            "name": "vmssExtensionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the VM scale set extension."
+          },
+          {
+            "name": "extensionParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VirtualMachineScaleSetExtensionUpdate"
+            },
+            "description": "Parameters supplied to the Update VM scale set Extension operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VirtualMachineScaleSetExtension"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
As this is a long running operation, is it better to return 202 instead of 200?

Reply:
I see that for all the other existing resources (VM, VMSS, AvSet, Image..) as well, only 200 (OK) is specified in the spec.